TUTORIAL.md

```diff
@@ -360,26 +360,22 @@ First, define where CesiumJS is. This tutorial uses the source code, so webpack
    import {
      Ion,
      Viewer,
-     createWorldTerrain,
-     createOsmBuildings,
+     Terrain,
+     createOsmBuildingsAsync,
      Cartesian3,
      Math,
    } from "cesium";
    import "cesium/Widgets/widgets.css";
    import "../src/css/main.css";
 
-   // Your access token can be found at: https://cesium.com/ion/tokens.
-   // This is the default access token
-   Ion.defaultAccessToken =
-     "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJqdGkiOiJlYWE1OWUxNy1mMWZiLTQzYjYtYTQ0OS1kMWFjYmFkNjc5YzciLCJpZCI6NTc3MzMsImlhdCI6MTYyNzg0NTE4Mn0.XcKpgANiY19MC4bdFUXMVEBToBmqS8kuYpUlxJHYZxk";
-
    // Initialize the Cesium Viewer in the HTML element with the `cesiumContainer` ID.
    const viewer = new Viewer("cesiumContainer", {
-     terrainProvider: createWorldTerrain(),
+     terrain: Terrain.fromWorldTerrain(),
    });
 
    // Add Cesium OSM Buildings, a global 3D buildings layer.
-   viewer.scene.primitives.add(createOsmBuildings());
+   const buildingTileset = await createOsmBuildingsAsync();
```

Comment:
iirc webpack 4 wasn't happy with a top level `await`. It may be better to do the `.then()` format from that example so it at least works for both 4 and 5?

Reply:
I think generally we want to show async/await where possible. This tutorial specifically uses Webpack 5+, so I think we're good here.